### PR TITLE
Updates to run on Tower 3.6 with PostgreSQL 10

### PR DESCRIPTION
**Pull request by rgerardi**

### CODE REVIEW

**Comment by aoyawale** [comment]
hello, can you fix the conflicts? so we can test this.

File: roles/samdoran.pgsql-replication/tasks/master.yml [hunk 1]
```diff
@@ -1,26 +1,59 @@
...
-    line: 'host    replication    {{ pgsqlrep_user }}  {{ item }}/32 trust'
-  with_items: "{{ pgsqlrep_replica_address }}"
-  notify: restart postgresql
+  - name: Add trust in pg_hba.conf
+    lineinfile:
+      state: present
+      dest: "{{ pgsqlrep_data_path }}/pg_hba.conf"
+      regexp: 'host.*replication.*{{ item }}/32.*trust'
+      line: 'host    replication    {{ pgsqlrep_user }}  {{ item }}/32 trust'
+    with_items: "{{ pgsqlrep_replica_address }}"
```
Comment (aoyawale) [inline roles/samdoran.pgsql-replication/tasks/master.yml:27]: can this be changed to loop?

File: roles/samdoran.pgsql-replication/tasks/master.yml [hunk 2]
```diff
@@ -1,26 +1,59 @@
...
+- name: apply master configuration block
+  block:
+  - name: Configure master server
+    lineinfile:
+      state: present
+      backrefs: yes
+      dest: "{{ pgsqlrep_data_path }}/postgresql.conf"
+      regexp: "{{ item.apply.regexp }}"
+      line: "{{ item.apply.line }}"
+    with_items: "{{ pgsqlrep_postgres_conf_lines }}"
```
Comment (aoyawale) [inline roles/samdoran.pgsql-replication/tasks/master.yml:10]: can this be changed to loop?

File: roles/samdoran.pgsql-replication/tasks/master.yml [hunk 3]
```diff
@@ -1,26 +1,59 @@
...
+- name: revert master configuration block
+  block:
+  - name: Configure master server
+    lineinfile:
+      state: present
+      backrefs: yes
+      dest: "{{ pgsqlrep_data_path }}/postgresql.conf"
+      regexp: "{{ item.revert.regexp }}"
+      line: "{{ item.revert.line }}"
+    with_items: "{{ pgsqlrep_postgres_conf_lines }}"
```
Comment (aoyawale) [inline roles/samdoran.pgsql-replication/tasks/master.yml:40]: same as above

File: roles/samdoran.pgsql-replication/tasks/master.yml [hunk 4]
```diff
@@ -1,26 +1,59 @@
...
+  #     state: absent
+  #   become: True
+  #   become_user: postgres
+
+  - name: Add trust in pg_hba.conf
+    lineinfile:
+      state: absent
+      dest: "{{ pgsqlrep_data_path }}/pg_hba.conf"
+      regexp: 'host.*replication.*{{ item }}/32.*trust'
+    with_items: "{{ pgsqlrep_replica_address }}"
```
Comment (aoyawale) [inline roles/samdoran.pgsql-replication/tasks/master.yml:57]: same as above

File: roles/samdoran.pgsql-replication/tasks/pgsql_version.yml [hunk 5]
```diff
@@ -1,13 +1,19 @@
-- name: Get PGSQL version
-  command: psql --version
+- name: Get PGSQL version 10.x
+  command: "{{ pgsqlrep_bash }} psql --version"
```
Comment (aoyawale) [inline roles/samdoran.pgsql-replication/tasks/pgsql_version.yml:2]: will have to check if we can use this module https://docs.ansible.com/ansible/latest/modules/postgresql_info_module.html#postgresql-info-module to get the version from there.

File: roles/samdoran.pgsql-replication/tasks/master.yml [hunk 6]
```diff
@@ -1,26 +1,59 @@
...
 
-- name: Add trust in pg_hba.conf
-  lineinfile:
-    state: present
-    dest: "{{ pgsqlrep_data_path }}/pg_hba.conf"
-    regexp: 'host.*replication.*{{ item }}/32.*trust'
-    line: 'host    replication    {{ pgsqlrep_user }}  {{ item }}/32 trust'
-  with_items: "{{ pgsqlrep_replica_address }}"
-  notify: restart postgresql
+  - name: Add trust in pg_hba.conf
```
Comment (aoyawale) [inline roles/samdoran.pgsql-replication/tasks/master.yml:21]: there's a module to modify the pg_hab.conf now https://docs.ansible.com/ansible/latest/modules/postgresql_pg_hba_module.html#postgresql-pg-hba-module wonder if we can use it and work fine between versions.

File: roles/samdoran.pgsql-replication/tasks/master.yml (hunk 1 above)
Comment (mpetrive-rh) [reply roles/samdoran.pgsql-replication/tasks/master.yml:27]: will update when I merge

File: roles/samdoran.pgsql-replication/tasks/master.yml (hunk 2 above)
Comment (mpetrive-rh) [reply roles/samdoran.pgsql-replication/tasks/master.yml:10]: will update when I merge

File: roles/samdoran.pgsql-replication/tasks/master.yml (hunk 3 above)
Comment (mpetrive-rh) [reply roles/samdoran.pgsql-replication/tasks/master.yml:40]: will update when I merge

File: roles/samdoran.pgsql-replication/tasks/master.yml (hunk 6 above)
Comment (mpetrive-rh) [reply roles/samdoran.pgsql-replication/tasks/master.yml:21]: Feature enhancement. Opened

**Pull request merged**